app/views/env_vars/index.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= render "apps/heading", app: @app, tab: "env_vars" %>
 <div class="columns">
   <div class="column is-8">
+    <div class="notification is-success is-hidden" data-role="sync-notification">
```

Comment:
Do we also automaticly refresh once those env vars are added? Since this message indicate we do